src/main.py

```diff
@@ -1848,11 +1848,12 @@ def _candidate_is_sync_with_primary(self, db_state, switchover_candidate):
                 continue
             if replica.get('application_name', '') != helpers.app_name_from_fqdn(switchover_candidate):
                 continue
-            replay_lag = replica.get('replay_location_diff', -1)
-            if replay_lag != 0:
+            replay_lag = replica.get('replay_lag', -1)
```

Comment:
`replica`, which is a single record from `get_replics_info()` call, does not have `replay_lag` field, it is spelled as `replay_lag_msec`: https://github.com/yandex/pgconsul/blob/main/src/pg.py#L327